### ISSUE TITLE
bug: warmup probe (line 291) + drafter ORT init (line 323) lack timeouts — same class as #1419

### ISSUE THREAD

**Issue by wordingone** [wordingone/gemma-architect#1420]
## Context

Same issue class as #1419 (post-drafter probe lacking timeout). Two additional paths in `model-worker.ts` have the same vulnerability.

## Path 1 — Warmup probe (line 291)

```typescript
await (_model as any).generate({ ...inputs, max_new_tokens: 8, do_sample: false });
```

- Runs on all non-recycle boots (`!noWarmup`, no cold-cache guard)
- 8 tokens with ~1300-token system prompt context
- No timeout — if the WebGPU generate() call hangs (e.g., GPU queue contention), boot is stuck indefinitely between `workerReady=true` and `bootComplete=false`

**Observed:** soft reload on 2026-05-22 with old code (pre-#1417) stuck in this exact state — `workerReady=true, bootComplete=false`, state=booting indefinitely. Root cause: the reload loaded old code where the post-drafter probe (#1419) was unfixed, AND the warmup probe likely encountered the same GPU queue issue during drafter ORT init.

## Path 2 — Drafter ORT init (line 323)

```typescript
_drafterSession = await ort.InferenceSession.create(drafterBuf, {
  executionProviders: ["webgpu", "wasm"],
  preferredOutputLocation: { logits: "cpu", proj_state: "cpu" },
});
```

- Runs on every boot (regardless of cold/warm)
- WebGPU shader compilation for ORT ops — can take 10-25 min on this hardware
- No timeout — if compilation hangs (GPU queue issue), boot never completes

## Fix options

**Option A (per-path):** Wrap each in `Promise.race([..., new Promise<void>(r => setTimeout(r, N_MS))])` like #1417:
- Warmup probe (line 291): `Promise.race([generate(...), timeout(60_000)])` — 60s; slower than post-drafter probe but still bounded
- Drafter ORT init (line 323): `Promise.race([InferenceSession.create(...), timeout(120_000)])` — 2min; if timed out, post `drafter-error` and continue

**Option B (boot-watchdog):** Add a top-level watchdog in `handleInit` that detects zero progress for N minutes and forces a synthetic `boot-complete` with error state. Cleaner architecture but larger change.

## Affected lines

- Warmup probe: `model-worker.ts` line 291 (or nearest — check current HEAD)
- Drafter ORT init: `model-worker.ts` line 323 (or nearest)

## Links

- Same class as: #1419
- Post-drafter probe fix: #1417
- Gate-discipline protocol note: any PR touching model-worker boot critical path must cite cold-cache contention scenario (per Leo's gate-discipline update after #1412)
